package.json

```diff
@@ -26,13 +26,13 @@
     "css-loader": "^0.28.0",
     "ts-loader": "^2.0.3",
     "tfx-cli": "^0.4.5",
-    "typescript": "^2.2.2",
+    "typescript": "2.3.4",
     "typings": "^2.1.0",
     "uglifyjs-webpack-plugin": "^0.4.2",
     "copy-webpack-plugin": "^4.0.1"
   },
   "dependencies": {
-    "vss-web-extension-sdk": "^2.114.0"
+    "vss-web-extension-sdk": "2.114.0"
```

Comment:
Caret will download the latest compatible version and I have seen build issues with latest vss sdk and typescript. This is just to be safe